layers/+tools/eaf/packages.el

```diff
@@ -236,20 +235,19 @@
         (lambda (prompt)
           (if (derived-mode-p 'eaf-mode)
               (pcase eaf--buffer-app-name
-                ((or
-                  (and "browser"
-                       (guard (string= (eaf-call-sync "call_function" eaf--buffer-id "is_focus") "True")))
```

Comment:
```
(guard (string= (eaf-call-sync "call_function" eaf--buffer-id "is_focus") "True")))
```
to
```
(guard (not (string= (eaf-call-sync "call_function" eaf--buffer-id "is_focus") "True"))))
```

Should fix this.

Reply:
I guess you mean I should just add the `not` in the original code (before the PR). Anyway, I have tried that but that also does not work (ESC does exit insert mode).

Reply:
Ah sorry... I think it does work, there is another reason why it seems broken. I don't know exactly how eaf reparenting works, but somehow (on Ubuntu) the browser window seems not to get well embedded in Emacs, but instead it opens in its own separate window. But from its position it looks to be embedded in Emacs, however when I select the browser window, that window gets active instead of the Emacs window (so that the keybindings don't seem to work). On Fedora it all seems to work fine (I am switching a little between Fedora and Ubuntu, because `mit-scheme` does not build in Fedora. Anyway.. this makes things a little more confusing.)

Reply:
Congrats with your new role in the Spacemacs team!

Reply:
Key binding issue aside, I've trouble with `SPC m d` for toggling dark-mode.
P.S. EAF never fully worked on my computer so I cannot fully test it.